File: golang/nb.go
package main

import (
  "fmt"
)


func main() {

  var firstName string
  var lastName string

  fmt.Println("What is your first name ?")
  fmt.Scanln(&firstName)
  fmt.Println("What is your last name ?")
  fmt.Scanln(&lastName)

  if firstName == "Linus" && lastName == "Zhang" {
    fmt.Println("> 牛逼!")
  } else {
    fmt.Println("> 不牛逼!")
  }

}

File: java/nb.java
import java.util.Scanner;

public class nb {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("What is your first name?");
        String firstName = scanner.nextLine();
        System.out.println("What is your last name?");
        String lastName = scanner.nextLine();
        System.out.println(">" + (firstName.equals("Linus") && lastName.equals("Zhang") ? "牛逼" : "不牛逼"));
    }
}
